np.sh

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env sh
+
+if test -n "$(git status --porcelain)"; then
```

Comment:
I don't know the wanted / expected behavior either. It was just an observation I made because it differs from previous behavior, it would be good with input from @sindresorhus on this matter.